src/Tribe/Generator/Event.php

```diff
@@ -80,6 +81,39 @@ public function random_event_data( $fromDate, $toDate ) {
 			'post_status'        => 'publish',
 			'tribe_test_data_gen'=> '1'
 			];
+
+		if( $is_virtual ) {
+		    $random_event_data += [
+                '_tribe_events_is_virtual'                  => 'yes',
+                '_tribe_events_virtual_embed_video'         => 'yes',
+                '_tribe_events_virtual_linked_button'       => 'yes',
+                '_tribe_events_virtual_show_embed_at'       => 'immediately',
+                '_tribe_events_virtual_show_on_event'       => 'yes',
+                '_tribe_events_virtual_show_on_views'       => 'yes',
+                '_tribe_events_virtual_rsvp_email_link'     => 'yes',
+                '_tribe_events_virtual_ticket_email_link'   => 'yes'
+            ];
+		    
+		    if ( mt_rand(0,1) ) {
+                $random_event_data += [
+                    '_tribe_events_virtual_url'                 => 'https://www.youtube.com/watch?v=W74FxZwhisM',
```

Comment:
What, not https://www.youtube.com/watch?v=dQw4w9WgXcQ ?

Reply:
LOL! No, this extension is now public facing, no Rick Rolling.